sbi/utils/sbiutils.py

```diff
@@ -31,20 +32,59 @@ def clamp_and_warn(name: str, value: float, min_val: float, max_val: float) -> f
     return clamped_val
 
 
-class Standardize(nn.Module):
+def standardizing_transform(
+    batch_t: Tensor, min_std: float = 1e-7
```

Comment:
I think I mentioned this a while ago, but I think the default should be lower, e.g. 1e-14.

The lower default would have to be passed down from the inference classes though.

Reply:
I changed the default to 1e-14. I think it's a sane value and I can't really see scenarios where users would want to change it -- which is why I'm not exposing it to the inference classes. If you have a use-case/strong preference, let me know and I'll expose it